lib/Phactory/Utils.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace Phactory;
+
+class Utils {
+  public static function getValueOrReturn($value) {
+    if(is_object($value) && is_callable($value)
```

Comment:
why test for is_object here? is_callable or instanceof Closure should be sufficient, no?